### DISCUSSION TITLE
password policy

### DISCUSSION THREAD

**Discussion by beKGB** [goharbor/harbor#19340] [Q&A]
I'm using the local harbor database, is it possible to define e.g. such rules? :

password_policy:
  min_length: 16
  min_lowercase: 2
  min_uppercase: 2
  min_digits: 2
  min_special_characters: 2
  change_password_days: 180
  history_length: 20

**Comment by wy65701436** [comment]
what policy are you pointing? the password of harbor user?

**Reply by beKGB** [reply]
yes , password for users logging into the harbor gui
